### PR TITLE
Milestone 5: Personal portfolio

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Personal portfolio milestone 5: Mobile version Footer section
In this milestone I was able to :
- Learn and implement forms in HTML
- Learn how to use formspree as a free form data handler
- Practice my Github flow knowledge
- Practice CSS layouts with flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -388,5 +389,34 @@ <h2>About me</h2>
             </div>
         </section>
     </main>
+    <footer class="footer">
+        <div class="footerHeader">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+
+        <div class="footerForm">
+            <form action="https://formspree.io/f/mgedowaj" method="POST">
```
Comment (lily-coder) [inline index.html:398]: - [x] [REQUIRED] As stated in the project requirements, you are to stick to the design as much as possible. Please fix the name and email input to match the original design and also edit the submit button by removing the black border and also edit the value of the submit button to match the original.😊😊
### Yours
footer1

### Figma Template
footer2

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @Nemwel-Boniface and @jssol , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good commit messages
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nemwel-Boniface) [reply index.html:398]: Hello, @lily-coder  thank you for the review.
- I have made the required changes. 
- I however do not know why it is showing a black border on the button. I have cross checked my styling and there is not black border.

Comment (mateo951) [reply index.html:398]: Please be sure to address the required changes asked by the previous reviewer. Sadly, your design still has the same issues as before 😥

image

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @Nemwel-Boniface and @jssol, 

Kindly make the requested changes by the previous reviewer 👏 You are really close to completing the project 🔥🙌 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nemwel-Boniface) [reply index.html:398]: Hello @mateo951  thank you for this review.
I have once again cross-checked my styling for the button and I have not found any set border of black. I have also noted that the black border only appears when viewed from a mobile device.
To remedy this, I have given the button class a border value of 0 and made the required changes and now there is no border appearing.

File: stylesheets/myWorks.css [hunk 2]
```diff
@@ -198,6 +198,7 @@ p {
   line-height: 24px;
   color: #fff;
   cursor: pointer;
+  border: 0;
```
Comment (Meltrust) [inline stylesheets/myWorks.css:201]: Good job correcting this button 💯

File: index.html [hunk 3]
```diff
@@ -388,5 +389,30 @@ <h2>About me</h2>
...
     </main>
+    <footer class="footer">
+        <div class="footerHeader">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+  
+        <div class="footerForm">
+            <form action="https://formspree.io/f/mgedowaj" method="POST">
+                <input class="ftext" type="text" id="Full-Name" name="name" placeholder="Full name" maxlength="30" required>
+                <input class="femail" type="email" id="Email" name="_replyto" placeholder="Email address" required>
```
Comment (Meltrust) [inline index.html:400]: - [ ] Please kindly correct these form fields as they have not been corrected as the previous reviewers requested.  I tested your project on Windows, and Linux, using Chrome and Firefox.  

Your project:

image

The Figma:

image

You can use the properties given in the template in order to figure out how to do it:

image

Let's correct these form fields so your project looks like the given design 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Nemwel-Boniface, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @Nemwel-Boniface  👋,

_This is an additional review as a result of the second opinion request._


I can see You focused on the requested change on the button alone and not for the name and email input which still remained the same, kindly note that it is a requirement for your design to match the given Figma design please make the necessary changes and we should approve manually 👍 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Pull request merged**